pkgs/development/python-modules/manticore/default.nix

```diff
@@ -0,0 +1,111 @@
+{ lib
+, buildPythonPackage
+, capstone
+, crytic-compile
+, fetchFromGitHub
+, pkgs
+, ply
+, prettytable
+, pyelftools
+, pyevmasm
+, pysha3
+, pytestCheckHook
+, pyyaml
+, rlp
+, setuptools
+, stdenv
+, unicorn
+, wasm
+, yices
+}:
+
+buildPythonPackage rec {
+  pname = "manticore";
+  version = "0.3.4";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "manticore";
+    rev = version;
+    sha256 = "0i9653013wvnj6y7f7i7vyj97250wd1qspr9ylpb1pnhi3c23gi0";
+  };
+
+  propagatedBuildInputs = [
+    crytic-compile
+    ply
+    prettytable
+    pyevmasm
+    pysha3
+    pyyaml
+    rlp
+    setuptools
```

Comment:
```suggestion
```
Probably redundant, I think default ``buildPythonPackage`` adds that.

Reply:
that seems correct, removed